### DISCUSSION TITLE
Adding ellipses to PCA analysis

### DISCUSSION THREAD

**Discussion by LaFra7** [vegandevs/vegan#504] [Q&A]
Hi all,

I perfermed a PCA using the rda function of the vegan package. I generated the plot using ggplot and I would like to plot also the ellipses. How can I do that?

Here is my code:

```
plot_PCA <- ggplot() +
  geom_point(data = PCAscores, aes(x = PC1, y = PC2, color = mydata$Species)) +
  scale_color_manual(values = pal) +
  geom_vline(xintercept = c(0), color = "grey70", linetype = 2) +
  geom_hline(yintercept = c(0), color = "grey70", linetype = 2) +
  geom_segment(data = PCAvect, aes(x = 0, y = 0, xend = PC1, yend = PC2), arrow = arrow(length = unit(0.2, "cm"))) +
  geom_text(data = PCAvect, aes(x = PC1, y = PC2, label = rownames(PCAvect))) +
  clean_background +
  labs(x = "PC1",
       y = "PC2",
       title = "Principal Components Analysis") 
plot_PCA
```

**Comment by gavinsimpson** [comment]
What are "the ellipses"? If you mean you want to add ggplot2 equivalents of `ordiellipse()` to you plot, please state so specifically. And this really isn't the place for questions about ggplot2... You could use the Discussions tab, but you will get more eyes on your problem by posting in a more appropriate venue, like on StackOverflow (I'd be surprised if this hadn't already been asked and answered there)

**Reply by LaFra7** [reply]
Yes, ordiellipse was what I meant. I am sorry for the mistake, I have seen you already moved my issue in discussion, is it right or should I do something?

Thank you

**Reply by gavinsimpson** [reply]
If you only want me to look at this (in which don't count on me having time to do so any time soon) then you don't need to do anything. If you want more eyes on the question, consider posting it in a more public/visible forum, like StackOverflow, as I suggested in my original reply. Before you do that i would search for similar questions there as I'd be surprised if you were the first person to want to do this and I even recall someone asking about this before...

**Reply by LaFra7** [reply]
Perfect, thank you very much!
